app/messages/messages/broker.rb

```diff
@@ -31,6 +31,9 @@ def connect!
       instantiate_exchange
       declare_queue
       bind_queue
+    rescue StandardError => e
+      Rails.logger.error("Cannot connect with RabbitMQ.")
+      ExceptionNotifier.notify_exception(e)
     end
```

Comment:
The only place I can see us initialise a RabbitMQ connection is in
https://github.com/sanger/labwhere/blob/b81de344ea37f5d9f4e85d518464082cf3b4f239/app/models/broker.rb#L7-L7
This means that if we fail to connect at start up, we will need to restart labwhere when bunny is restored.

Reply:
Also, may be worth sticking `e.message` in the log, just in case it's another error being rescued (such as an authentication error). The information would be in the email, but that won't help in, for example, development mode.

Reply:
We still only appear to attempt a connection on app start up. Seeing as '#create_connection' begins by checking to see if a connection already exists, we could just do this as part of publish. (Or rescue the lack of connection and retry once)